help_function/src/onedpl_test_arg_index_input_iterator.cpp

```diff
@@ -0,0 +1,502 @@
+// ====------ onedpl_test_arg_index_input_iterator.cpp---------- -*- C++ -*
+// ----===////
```

Comment:
why change a line.

Reply:
Ah, this is a mistake.  I'll correct it.  Result of clang-format.

Reply:
Whoops, it seems it was already merged...
@tangjj11 @ShengchenJ would you like me to open another PR to correct this?